index.js

```diff
@@ -6,6 +6,28 @@ function Eris(token, options) {
     return new Client(token, options);
 }
 
+function toString() {
+	var str;
+	Object.keys(Eris).filter(eris_prop => typeof Eris[eris_prop] === "function" && this instanceof Eris[eris_prop]).sort((a, b) => Eris[b].prototype instanceof Eris[a] ? 1 : 0).forEach(eris_prop => {
+		if(!str) {
+		    var ID = "";
+			if(this.id) {
+			    ID = " " + this.id;
+			}else if(this.user) {
```

Comment:
I doubt something like that would make any major difference but sure I'll change that to match.